tests/ui/lint/lint-overflowing-ops.rs

```diff
@@ -242,26 +242,44 @@ fn main() {
     let _n = 1u128 / 0; //~ ERROR: this operation will panic at runtime
     let _n = &(1u128 / 0); //~ ERROR: this operation will panic at runtime
 
+    let _n = 1usize / 0; //~ ERROR: this operation will panic at runtime
+    let _n = &(1usize / 0); //~ ERROR: this operation will panic at runtime
+
     let _n = 1i8 / 0;   //~ ERROR: this operation will panic at runtime
     let _n = &(1i8 / 0);   //~ ERROR: this operation will panic at runtime
+    let _n = i8::MIN / -1; //~ ERROR: this operation will panic at runtime
+    let _n = &(i8::MIN / -1); //~ ERROR: this operation will panic at runtime
+    //~^ERROR: evaluation of constant value failed
```

Comment:
Something is going wrong with promotion here leading to a second error message; I'll investigate this separately.

Reply:
Found it: https://github.com/rust-lang/rust/pull/121515